README.md

```diff
@@ -59,8 +59,5 @@ CUDA_VISIBLE_DEVICES=0 FLOCK_API_KEY="<your-api-key>" python validate.py validat
 If you wish to continuously receive task assignments, you should use the following command:
 
 ```bash
-CUDA_VISIBLE_DEVICES=0 FLOCK_API_KEY="<your-api-key>" HF_TOKEN="your-hf-token-to-access-gated-based-models" python validate.py loop \
---task_id <list-of-task-ids> \ # For example: "2,3" OR "2"
---validation_args_file validation_config.json.example \
---auto_clean_cache False # whether to automatically clean up non base models cache
+./monitor.sh --hf_token your_hf_token --flock_api_key your_flock_api_key --task_id your_task_id --validation_args_file validation_config.json.example --auto_clean_cache False
```

Comment:
`bash xxx.sh`
so that the script file does not need execute permission; read permission